src/Stratis.Bitcoin.IntegrationTests/Utilities/Extensions/CoreNodeExtensions.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using NBitcoin;
+using Stratis.Bitcoin.Features.Consensus;
+using Stratis.Bitcoin.Features.Consensus.Interfaces;
+using Stratis.Bitcoin.IntegrationTests.EnvironmentMockUpHelpers;
+
+namespace Stratis.Bitcoin.IntegrationTests.Utilities.Extensions
+{
+    public static class CoreNodeExtensions
+    {
+        public static Money CalculateProofOfWorkReward(this CoreNode node, int blockCount)
```

Comment:
This still won't work will it?  The reward is related to the blockheight - is that used in the implementation? I can't see it?